bootstrap-salt.sh

```diff
@@ -2783,7 +2805,7 @@ install_debian_git_post() {
             if [ -f "${_SALT_GIT_CHECKOUT_DIR}/debian/salt-$fname.init" ]; then
                 __copyfile "${_SALT_GIT_CHECKOUT_DIR}/debian/salt-$fname.init" "/etc/init.d/salt-$fname"
             else
-                __fetch_url "/etc/init.d/salt-$fname" "http://anonscm.debian.org/cgit/pkg-salt/salt.git/plain/debian/salt-${fname}.init"
+                __fetch_url "/etc/init.d/salt-$fname" "https://anonscm.debian.org/cgit/pkg-salt/salt.git/plain/debian/salt-${fname}.init"
```

Comment:
@jfindlay In this particular case the bootstrap fetches plain SystemV-style initscripts for Debian 7. Unfortunately, they aren't present in current develop branch of `saltstack/salt`repo. There are even no `pkg/debian` directory.

That's why I suggest to include the init files to the `saltstack/salt` as well.